data-plane-framework/draft-ietf-detnet-data-plane-framework.xml

```diff
@@ -240,55 +242,55 @@
        <t>
          The service sub-layer provides additional support beyond the
          connectivity function of the forwarding sub-layer. An example
-         of this is Packet Replication, Elimination, and Ordering (PREOF)
+         of this is Packet Replication, Elimination, and Ordering
          functions see <xref target="PREOF"/>.
        </t>
        <t>
-         The method of instantiating each of the layers is specific to the
-         particular DetNet data plane method. There may be more than one approach
-         that is applicable to a given bearer network type.
+         The method of instantiating each of the layers is specific to the 
+		 particular DetNet data plane method, and more than one approach may 
+		 be applicable to a given bearer network type.
        </t>
 
       <section title="Data Plane Characteristics" anchor="sec_dp_char">
         <t>
-	There are two major characteristics to the data plane:
+	There are two major characteristics to the data plane: the technology and 
+	the encapsulation, as discussed below.
         </t>
+		<section title="Data Plane Technology" anchor="sec_dp_tech">
          <t>
-         <list style='numbers'>
-         <t>
-         Data plane technology: The DetNet data plane is provided by the DetNet 
-		 service and forwarding sub layers.  The DetNet service sub-layer

```

Comment:
I think the sentence "The DetNet data plane is provided by the DetNet service and forwarding sub layers. "  Should not be dropped, suggest adding back as the frst sentence in the new section.

Reply:
This was already clarified earlier in the draft. Yes, it is valid, but a already said. That is the reason for the removal.

Reply:
In reading the section, I think the sentence still has value,

Reply:
I was the one who suggested that it felt redundant in the context of reading the draft end-to-end.  I don't object to putting it back in.